### ISSUE TITLE
Vue language server support 

### ISSUE THREAD

**Issue by bajrangCoder** [mkslanc/ace-linters#144]
The `@vue/language-server` doesn't show completions, though other features seem to be working fine. In the console, there are warnings indicating that certain LSP specifications are not supported by the client but are used by the server, such as:

```
textDocument.completion.completionItem.insertReplaceSupport
```

I have tested this with the latest version of `ace-linter` and your LSP WebSocket bridge.

**Comment by bajrangCoder** [comment]
I have also used same config. Is completion working in script tag ? Or any vue specific completion.

I know css, html completion working perfectly

**Comment by mkslanc** [comment]
@bajrangCoder Which vue language server, typescript version do you use? Also which configs if there are any. I couldn't manage it to show anything inherited from ts service, so no diagnostics, completions, hover and other stuff inside script tags.

**Comment by mkslanc** [comment]
Seems I managed to make it work -> you just need to turn off hybrid mode of vue (not even sure what it is) like this -
```javascript
manager.registerServer("vue", {
    module: () => import("ace-linters/build/language-client"),
    modes: "vue",
    type: "socket",
    socket: new WebSocket("ws://127.0.0.1:3080/vue"),
    initializationOptions: {
        typescript: {
            tsdk: "node_modules/typescript/lib", //path to typescript server
        },
        vue:  {
            hybridMode: false
        },
    }
});
```

**Comment by bajrangCoder** [comment]
It doesn't seems , as I have already tried

**Comment by bajrangCoder** [comment]
> Which vue language server, typescript version do you use? Also which configs if there are any. I couldn't manage it to show anything inherited from ts service, so no diagnostics, completions, hover and other stuff inside script tags.

Latest vue server, latest typescript version and the same config as yours

**Comment by bajrangCoder** [comment]
I think this issue is related to typescript language server, not with the ace linter

**Comment by bajrangCoder** [comment]
Hmm, these starts working with latest ace-linter version.

**Comment by bajrangCoder** [comment]
I have one offtopic question but related to this project:
There few wasm based formatter available for languages like lua, clang, zig, go, etc.
So it is good to integrate those things directly in ace-linters or through extensions.

Second thing is python ruff linter got some updates, as now it supports formatter etc(so good to update it)
